modules/file-system/pre-existing-network-storage/outputs.tf

```diff
@@ -24,3 +24,55 @@ output "network_storage" {
     mount_options = var.mount_options
   }
 }
+
+locals {
```

Comment:
I did not know you could put locals blocks in outputs.tf... Or at least never thought to. 
I like it, it's a lot cleaner keeping anything output related only with the output.

Reply:
I am pretty sure that with terraform, anything can go anywhere. I personally agree that it is clearest to keep this locals block in output but also am not too opinionated.